### ISSUE TITLE
Enhancement: show the native languages I

### ISSUE THREAD

**Issue by uhuklein** [LangCorrect/server#297]
When we correct an entry we can see the gender used in the text and the language skills level. It would be great to add a mention of the native languages of the writer. This way we could truly adapt our corrections and comments.

**Comment by danielzeljko** [comment]
We already have a language level for this. Can you provide some insights into how knowing the native language(s) can help adapt the corrections and comments?

We can add the native language(s) here:

![image](https://github.com/LangCorrect/server/assets/115326106/0bccf6dc-feef-44ad-8dae-32488a832183)

**Comment by keyxcode** [comment]
I can see this being helpful in situations where the corrector speaks or has some knowledge of the writer's native language. This may help them detect systematic errors such as the tendency of certain languages to omit the English articles.

**Comment by uhuklein** [comment]
Exactly, it can help recognize some schemes and give a better feedback to the writer. It can help to detect false friends between the native and target language and explain the nuances. Secondly, if I know the writer's native language, I can provide some translations in case of a tricky mistake. It is specially useful if the writer is a beginner and cannot spend time on every comment in a language they still not master.

As a always start correcting in the target is language, I usually realise the need of this knowledge while I am already correcting and I cannot save my correction as a draft before clicking on the writer's profile.

**Comment by danielzeljko** [comment]
I'm all for this. Thanks for the insights and discussion! 

related #140

**Comment by ee22abc** [comment]
I agree very much that we want to see the version that the poster wrote in his/her native language while correcting it. Even though it's not always the case that the corrector knows the poster's native language, some are adding their native language version. If we cannot see it beforehand, then we are just wasting the efforts.

**Comment by ee22abc** [comment]
I found the button "Post Overview" at the top right and this shows the native version text, but I have to say it is not made very convenient. 
It would be good if the text appears between the top blue bar and the correction area, and stay there when the window is scrolled.

We should be okay to sacrifice some correction space since we only need to see one sentence at a time when correcting.
